builtin/providers/aws/resource_aws_rds_cluster.go

```diff
@@ -156,6 +184,18 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
 		createOpts.AvailabilityZones = expandStringList(attr.List())
 	}
 
+	if v, ok := d.GetOk("backup_retention_period"); ok {
```

Comment:
This is fine to leave, but fwiw anything with a `Default` specified will always return `true` for `ok` in `d.GetOk`